.github/workflows/rosa-cluster-auto-provision-on-schedule.yml

```diff
@@ -72,8 +72,39 @@ jobs:
       skipAuroraDeletion: true
     secrets: inherit
 
-  keycloak-deploy-active-active:
+  keycloak-deploy-with-external-infinispan:
     needs: keycloak-undeploy-with-persistent-sessions
+    name: ROSA Scheduled Create cluster with External Infinispan Feature
+    if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-benchmark'
+    uses: ./.github/workflows/rosa-multi-az-cluster-create.yml
+    with:
+      clusterPrefix: gh-keycloak # ${{ env.CLUSTER_PREFIX }} -- unfortunately 'env.' doesn't work here
+      enableExternalInfinispanFeature: true
+      createCluster: false
+    secrets: inherit
+
+  run-scaling-benchmark-with-external-infinispan:
```

Comment:
We're going to need a way to differentiate benchmark runs executed with "remote-cache" and normal A/P. This is also required for Active/Active. I've created https://github.com/keycloak/keycloak-benchmark/issues/862 to address this in a separate PR.

Reply:
I added a field: https://github.com/keycloak/keycloak-benchmark/blob/main/provision/common/Taskfile.yaml#L85

Reply:
It appears I'm blind, this just applies to A/A then :tada: